contextualize_pnid_files.ipynb

```diff
@@ -63,7 +63,7 @@
    "source": [
     "# Get files with source\n",
     "source = \"some_source\"\n",
-    "files = client.files.list(source=source, limit=-1)"
+    "files = client.files.list(mime_type=\"application/pdf\", source=source, limit=-1)"
```

Comment:
This change is not reflected by the PR title. The PR title claims that you only fix getpass, so this belongs in another PR.